addons/hr_expense/wizard/hr_expense_split_wizard_views.xml

```diff
@@ -13,14 +13,13 @@
                             <field name="currency_id" column_invisible="True"/>
                             <field name="expense_id" column_invisible="True"/>
                             <field name="company_id" column_invisible="True"/>
-                            <field name="product_has_tax" column_invisible="True"/>
```

Comment:
I think this is breaking the stable policy. Imagine someone in a customization adding a readonly field based on the product_has_tax field, it will break when upgrading the module. So I would just make this line change only in the master forward port